### DISCUSSION TITLE
How to use `create gCal Event from Frontmatter` command??

### DISCUSSION THREAD

**Discussion by ghost** [YukiGasai/obsidian-google-calendar#232] [Q&A] [answered]
Hello
How can I use this command 'create gcal event from frontmatter'?

I used a translator because I was not good at English.
I'm sorry that the grammar may not be right.

The Google calendar plug-in makes managing your schedule more convenient.

when I'm using it very happy, and I trying understand this plug-in more
but unhappy 'create gCalEvent from Frontmatter' command operation is unknown.

This command appears to convert to an event note based on information from the frontmatter.

I want to use it because it looks very useful, but I don't have background knowledge, so I invested 5 hours and didn't understand.

how can i use this command?

**Comment by YukiGasai** [comment, marked as answer]
Hi, have you read the [documentation page](https://yukigasai.github.io/obsidian-google-calendar/EventNotes/FrontmatterActions) for the command?

**Reply by ghost** [reply]
OMG, I'm really embarrassed
Thank you for the information.
